### ISSUE TITLE
Error when exporting: org-hugo-paragraph: Symbol’s function definition is void: org-blackfriday--div-wrap-mayb

### ISSUE THREAD

**Issue by jackbaty** [kaushalmodi/ox-hugo#279]
## Actual Behavior

When exporting a heading (`C-c C-e H H`) I see the following error...

`org-hugo-paragraph: Symbol’s function definition is void: org-blackfriday--div-wrap-maybe`

## Expected Behavior

Heading should be exported to a markdown file.

## How to Reproduce the Issue

Open the sample content file, place the point in a heading, press `C-c C-e H H`

### Example Org File

I am using the [sample file](https://github.com/kaushalmodi/ox-hugo/blob/master/test/site/content-org/all-posts.org)

### Generated Markdown File or Error

`org-hugo-paragraph: Symbol’s function definition is void: org-blackfriday--div-wrap-maybe`

No markdown file is generated

## Ox-Hugo Debug Information

<details>
  <summary>Debug Info</summary>

# Debug information for `ox-hugo`

## Emacs Version

    GNU Emacs 26.1 (build 1, x86_64-apple-darwin14.5.0, NS appkit-1348.17 Version 10.10.5 (Build 14F2511))
     of 2018-05-30

## Org Version

    Org mode version 9.2.4 (9.2.4-1-g0d7d12-elpaplus @ /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/)

## Hugo Version

    =hugo= binary not found in PATH

### Org `load-path` shadows

**Warning**: Possible mixed installation of Org

    /usr/local/share/emacs/site-lisp/mu/mu4e/org-mu4e hides ~/Dropbox/Sync/lisp/org-mu4e
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-stan hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-stan
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-exp hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-exp
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-J hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-J
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-eshell hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-eshell
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-emacs-lisp hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-emacs-lisp
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-gnus hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-gnus
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-css hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-css
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-lob hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-lob
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-forth hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-forth
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-macs hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-macs
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-version hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-version
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-scheme hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-scheme
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-abc hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-abc
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-C hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-C
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-capture hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-capture
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-ref hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-ref
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-clojure hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-clojure
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-mouse hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-mouse
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-ledger hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-ledger
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-ctags hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-ctags
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-entities hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-entities
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-archive hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-archive
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-screen hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-screen
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-haskell hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-haskell
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-asymptote hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-asymptote
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-mhe hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-mhe
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-table hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-table
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-keys hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-keys
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-org hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-org
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-plot hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-plot
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-awk hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-awk
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-groovy hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-groovy
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-octave hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-octave
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-faces hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-faces
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-colview hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-colview
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-R hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-R
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-timer hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-timer
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-ebnf hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-ebnf
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-mobile hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-mobile
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-fortran hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-fortran
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-shell hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-shell
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-perl hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-perl
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-sqlite hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-sqlite
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-sed hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-sed
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-list hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-list
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-ruby hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-ruby
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-eval hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-eval
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-habit hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-habit
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-clock hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-clock
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-html hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-html
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-src hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-src
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-lisp hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-lisp
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-ditaa hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-ditaa
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-pcomplete hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-pcomplete
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-lint hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-lint
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-rmail hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-rmail
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-latex hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-latex
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-sass hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-sass
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-io hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-io
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-tangle hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-tangle
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-calc hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-calc
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-java hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-java
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-icalendar hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-icalendar
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-eww hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-eww
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-md hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-md
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-beamer hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-beamer
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-element hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-element
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-protocol hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-protocol
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-mscgen hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-mscgen
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-gnuplot hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-gnuplot
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-latex hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-latex
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-id hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-id
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-vala hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-vala
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-man hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-man
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-feed hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-feed
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-lua hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-lua
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-table hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-table
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-ocaml hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-ocaml
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-coq hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-coq
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-picolisp hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-picolisp
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-indent hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-indent
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-lilypond hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-lilypond
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-matlab hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-matlab
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-datetree hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-datetree
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-python hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-python
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-bbdb hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-bbdb
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-makefile hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-makefile
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-duration hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-duration
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-agenda hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-agenda
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-dot hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-dot
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-js hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-js
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-publish hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-publish
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-inlinetask hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-inlinetask
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-org hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-org
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-core hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-core
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-compat hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-compat
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-docview hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-docview
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-odt hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-odt
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-plantuml hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-plantuml
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-ascii hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-ascii
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-loaddefs hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-loaddefs
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-w3m hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-w3m
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-bibtex hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-bibtex
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-info hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-info
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-hledger hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-hledger
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-maxima hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-maxima
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org hides /Applications/Emacs.app/Contents/Resources/lisp/org/org
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-macro hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-macro
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-sql hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-sql
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-attach hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-attach
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-processing hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-processing
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ox-texinfo hides /Applications/Emacs.app/Contents/Resources/lisp/org/ox-texinfo
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-irc hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-irc
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-crypt hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-crypt
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-footnote hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-footnote
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/org-install hides /Applications/Emacs.app/Contents/Resources/lisp/org/org-install
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-comint hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-comint
    /Users/jbaty/.emacs.d/elpa/org-plus-contrib-20190624/ob-shen hides /Applications/Emacs.app/Contents/Resources/lisp/org/ob-shen

Study the output of `M-x list-load-path-shadows`.

## `ox-hugo` defcustoms

<table border="2" cellspacing="0" cellpadding="6" rules="groups" frame="hsides">

<colgroup>
<col  class="org-left" />

<col  class="org-left" />
</colgroup>
<tbody>
<tr>
<td class="org-left">org-hugo-section</td>
<td class="org-left">"posts"</td>
</tr>

<tr>
<td class="org-left">org-hugo-use-code-for-kbd</td>
<td class="org-left">nil</td>
</tr>

<tr>
<td class="org-left">org-hugo-preserve-filling</td>
<td class="org-left">t</td>
</tr>

<tr>
<td class="org-left">org-hugo-delete-trailing-ws</td>
<td class="org-left">t</td>
</tr>

<tr>
<td class="org-left">org-hugo-prefer-hyphen-in-tags</td>
<td class="org-left">t</td>
</tr>

<tr>
<td class="org-left">org-hugo-allow-spaces-in-tags</td>
<td class="org-left">t</td>
</tr>

<tr>
<td class="org-left">org-hugo-tag-processing-functions</td>
<td class="org-left">(org-hugo&#x2013;tag-processing-fn-replace-with-spaces-maybe org-hugo&#x2013;tag-processing-fn-replace-with-hyphens-maybe)</td>
</tr>

<tr>
<td class="org-left">org-hugo-auto-set-lastmod</td>
<td class="org-left">nil</td>
</tr>

<tr>
<td class="org-left">org-hugo-export-with-toc</td>
<td class="org-left">nil</td>
</tr>

<tr>
<td class="org-left">org-hugo-export-with-section-numbers</td>
<td class="org-left">nil</td>
</tr>

<tr>
<td class="org-left">org-hugo-front-matter-format</td>
<td class="org-left">"toml"</td>
</tr>

<tr>
<td class="org-left">org-hugo-default-static-subdirectory-for-externals</td>
<td class="org-left">"ox-hugo"</td>
</tr>

<tr>
<td class="org-left">org-hugo-external-file-extensions-allowed-for-copying</td>
<td class="org-left">("jpg" "jpeg" "tiff" "png" "svg" "gif" "mp4" "pdf" "odt" "doc" "ppt" "xls" "docx" "pptx" "xlsx")</td>
</tr>

<tr>
<td class="org-left">org-hugo-date-format</td>
<td class="org-left">"%Y-%m-%dT%T%z"</td>
</tr>

<tr>
<td class="org-left">org-hugo-paired-shortcodes</td>
<td class="org-left">""</td>
</tr>

<tr>
<td class="org-left">org-hugo-langs-no-descr-in-code-fences</td>
<td class="org-left">nil</td>
</tr>

<tr>
<td class="org-left">org-hugo-suppress-lastmod-period</td>
<td class="org-left">0.0</td>
</tr>

<tr>
<td class="org-left">org-hugo-front-matter-format</td>
<td class="org-left">"toml"</td>
</tr>
</tbody>
</table>

  

</details>

**Comment by kaushalmodi** [comment]
That's an odd error.. the `org-blackfriday--div-wrap-maybe` is indeed defined .. here:

https://github.com/kaushalmodi/ox-hugo/blob/df527928fc05acec8d426654f4e21b4e17d5f5dc/ox-blackfriday.el#L346

I cannot recreate the error (also Travis CI is successfully auto-generating the ox-hugo docs site).

- How are you adding `ox-hugo` to your Emacs setup?
- Are you using one of these styles? https://ox-hugo.scripter.co/doc/usage/
- If you uninstall/reinstall `ox-hugo` from Melpa and restart Emacs, does this issue go away?
- If above did not work, try opening `ox-blackfriday.el`, do `M-x eval-buffer` and then try exporting an Org post. Does that error still occur?

---

In summary, it looks like somehow `ox-blackfriday.el` did not get loaded on your system.

**Comment by jackbaty** [comment]
Thanks for the thorough response. This turned out to be totally my issue. I went to eval `ox-blackfriday.el` per your suggestion and noticed that I still had a very old copy of that file in my ~/lisp directory. Removing that old file seems to have cleared things up. 

Sorry for the noise and thanks for the help!

**Comment by kaushalmodi** [comment]
> Sorry for the noise

No worries!

>  and thanks for the help!

You are welcome :)
